### PR TITLE
sig-release: Add Carlos Tadeu Panato Jr. & Adolfo García Veytia as TLs

### PR DESCRIPTION
**Pull request by justaugustus**
From https://groups.google.com/a/kubernetes.io/g/release-managers/c/zzk3naZhS4A:

> @kubernetes/release-engineering,
> 
> After some discussion with the leads, we'd like to propose both @cpanato and @puerco as Technical Leads for SIG Release, and more specifically, the Release Engineering subproject.
> 
> They are both dedicated Release Managers and never fail to lend a helping hand to contributors or jump into unfamiliar territory to support the subproject.
> 
> We're opening this for discussion with Release Managers, before formally proposing this on the mailing list.

Signed-off-by: Stephen Augustus <foo@auggie.dev>

/assign @saschagrunert @hasheddan @LappleApple @jeremyrickard 
cc: @kubernetes/sig-release

### CODE REVIEW

**Review by hasheddan — APPROVED** [review]
/lgtm

Well deserved! :tada:

**Review by saschagrunert — APPROVED** [review]
LGTM 🥳

**Comment by justaugustus** [comment]
Note sent to k-dev: https://groups.google.com/g/kubernetes-dev/c/th2CftV0rOs
Lazy consensus timeout: Wednesday, June 9th at 4pm US ET

Assigning @kubernetes/steering-committee liaison:
/assign @dims

**Comment by dims** [comment]
YESSSS!!! congrats @cpanato and @puerco

**Comment by mrbobbytables** [comment]
/pony party
🎉

**Comment by Damans227** [comment]
@cpanato and @puerco Well deserved guys. Keep up the good work!

**Comment by jeremyrickard** [comment]
+1 and 
/honk +1

**Comment by jeremyrickard** [comment]
/lgtm

As well :)

**Comment by JamesLaverack** [comment]
huge +1
/lgtm

**Comment by spiffxp** [comment]
> more specifically, the Release Engineering subproject.

@kubernetes/steering-committee WDYT about adding a tech_leads field to subprojects?

Alternatively, could this be better expressed as promoting them to subproject owners?

**Comment by justaugustus** [comment]
> > more specifically, the Release Engineering subproject.
> 
> @kubernetes/steering-committee WDYT about adding a tech_leads field to subprojects?

This is a potential discussion point for https://github.com/kubernetes/community/issues/5722.

> Alternatively, could this be better expressed as promoting them to subproject owners?

@cpanato and @puerco have leadership experience across both Release Engineering and Release Team efforts, so a Tech Lead role is most appropriate for them.

**Review by xmudrii — APPROVED** [review]
+1 :tada:

/lgtm

**Comment by annajung** [comment]
+1 @cpanato @puerco, thank you both for everything you do!

**Comment by reylejano** [comment]
+1 well deserved 🎉

**Comment by jberkus** [comment]
+1

**Comment by tpepper** [comment]
lgtm 🎉

**Comment by mrbobbytables** [comment]
> > more specifically, the Release Engineering subproject.
> 
> @kubernetes/steering-committee WDYT about adding a tech_leads field to subprojects?
> 
> Alternatively, could this be better expressed as promoting them to subproject owners?

@spiffxp I think the plan is to move forward with enabling a subproject leads type field. (ref: https://github.com/kubernetes/community/issues/5722) The codifying and mechanics of it haven't been completed yet =/ E_TOO_MANY_*

**Comment by ameukam** [comment]
Huge +1!
/lgtm
/honk

**Comment by divya-mohan0209** [comment]
Huuge +1 @cpanato & @puerco Congratulations folks! This is so so well deserved 💜
/honk

**Comment by kikisdeliveryservice** [comment]
two of the very best! huge +1 :heart: :heart:

**Comment by justaugustus** [comment]
> Note sent to k-dev: https://groups.google.com/g/kubernetes-dev/c/th2CftV0rOs
> Lazy consensus timeout: Wednesday, June 9th at 4pm US ET

Hearing no objections from the SIG and with lazy consensus timeout achieved, @kubernetes/sig-release-leads would like to welcome @cpanato and @puerco to the leadership team!

**Comment by justaugustus** [comment]
cc: @kubernetes/steering-committee + @kubernetes/sig-contributor-experience-leads for root `/approve`.

**Comment by justaugustus** [comment]
Away we go!
/pony party
/honk

...but most importantly:
/hold cancel

**Comment by justaugustus** [comment]
Onboarding umbrella issue: https://github.com/kubernetes/sig-release/issues/1590

**Comment by danpopSD** [comment]
Love you both. Congrats!

**Pull request merged**
